persistence/types/converters.go

```diff
@@ -0,0 +1,149 @@
+package types
+
+import "github.com/pokt-network/pocket/shared/modules"
+
+func toPersistenceActor(actor modules.Actor) *Actor {
+	return &Actor{
+		Address:      actor.GetAddress(),
+		PublicKey:    actor.GetPublicKey(),
+		StakedAmount: actor.GetStakedAmount(),
+		GenericParam: actor.GetGenericParam(),
+	}
+}
+
+func ToPersistenceActors(actors []modules.Actor) []*Actor {
```

Comment:
Another case where cross module importing is introduced. (Used by p2p)

Reply:
This doesn't happen anymore